llvm/test/CodeGen/X86/subreg-fail.mir

```diff
@@ -0,0 +1,37 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 5
```

Comment:
Pre-commit this case to show where `MachineVerifier` complains when specifying `-verify-coalescing -enable-subreg-liveness=true`?